lockfiles/black

```diff
@@ -0,0 +1,442 @@
+// This lockfile was autogenerated by Pants. To regenerate, run:
```

Comment:
Pants doesn't care at all what file extension you use, but maybe helpful to call this `black.lock`? Or `black.jsonc` (JSON w/ comments)

Reply:
Since users shouldn't really care mess with these files either, does the extension make a difference? If it needs an extension, I'd go for `.lock`.

Reply:
It makes zero difference to Pants - only whatever naming convention makes sense for your repo